utils/python/CIME/SystemTests/eri.py

```diff
@@ -9,12 +9,13 @@ def __init__(self, caseroot, case):
         """
         initialize an object interface to the ERI system test
         """
-        SystemTestsCommon.__init__(self, caseroot, case)
+        SystemTestsCommon.__init__(self,  caseroot, case)
+        self._testname = "ERI"
 
     def run(self):
-        self._case.set_value("CONTINUE_RUN", False)
-        self._case.set_value("REST_OPTION","none")
-        self._case.set_value("HIST_OPTION","$STOP_OPTION")
-        self._case.set_value("HIST_N","$STOP_N")
```

Comment:
Were these commented out for debugging? If we don't need them at all, please delete.